extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerConfig.java

```diff
@@ -118,4 +118,9 @@
     @ConfigItem(defaultValue = "true")
     public Boolean logTraceContext;
 
+    /**
+     * Whether the jaeger should run in zipkin compatibility mode
+     */
+    @ConfigItem
+    public Optional<Boolean> zipkinCompatibilityMode;
```

Comment:
Any reason why an `Optional` instead of a `Boolean` with `defaultValue = "false"`?

Reply:
I changed to boolean